src/titlepane/TitlePane.ts

```diff
@@ -108,36 +132,19 @@ export default class TitlePane extends TitlePaneBase<TitlePaneProperties> {
 		}, [
 			v('div', {
 				'aria-level': headingLevel ? String(headingLevel) : null,
-				classes: this.classes(
-					closeable ? css.closeable : null,
-					css.title
-				).fixed(
-					closeable ? css.closeableFixed : null,
-					css.titleFixed
-				),
-				onclick: this._onTitleClick,
-				onkeyup: this._onTitleKeyUp,
+				classes: this.classes(css.title, ...this.getModifierClasses()).fixed(css.titleFixed, ...this.getFixedModifierClasses()),
 				role: 'heading'
 			}, [
-				v('div', {
+				v('button', {
```

Comment:
Why did this change to a button?

Reply:
I always thought it should be a button, since one of the basic a11y principles in our widget design is to use native elements whenever possible. I just took this widget pattern PR as a good place to change it. It also simplifies the code a ton, since we no longer need to handle focus and key events manually.